### DISCUSSION TITLE
Question About Messages.yml File

### DISCUSSION THREAD

**Discussion by MrApocalyptic** [GriefPrevention/GriefPrevention#800] [Q&A] [answered]
Hello, I was editing some of the messages in the messages.yml file and was wondering about a couple of the message. When you do /claimlist it shows this http://prntscr.com/s99iu8 I was wondering if theres a section in the file to edit the world part of it, and when an admin uses a stick to check if someone owns a block it shows this http://prntscr.com/s99iow, is there a way to edit the 22x21=462 part? Hopefully I'm not overlooking it, but thanks for taking time out of your day to read this :D

**Comment by RoboMWM** [comment, marked as answer]
I think there's a feature request for this.

No it's currently not modifiable.
